README.md

```diff
@@ -55,6 +55,149 @@ Edit the file `cdc-accelerator.json` in the project directory and add the `sourc
+To use them, simply write on the terminal
+
+```
+npm run <operation> -- -f <feature>
+```
+
+For example:
+
+```
+npm run init -- -f Schema
+```
+
+In this example the user is only going to run the feature Schema when running the operation Init, the feature name can be replaced by any other feature (Email Templates, WebScreenSet, PermissionGroup, WebSdk...).
+To show all the possible commands, the user can write simply
+
+```
+npx cdc help
+```
+
+This command will show all possible commands and options that the user can do, the output will be something like this:
+
+```
+Usage: npx cdc [options] [command]
+
+A development environment for SAP Customer Data Cloud that enables the use of modern tools, such as JavaScript and source control.
+
+Options:
+  -V, --version     output the version number
+  -h, --help        display help for command
+
+Commands:
+  start             Launch local server for testing using the preview functionality
+  setup             Setup a new project after this dependency is installed
+  init [options]    Reads the data from the cdc console of the sites configured
+  reset [options]   Deletes the local folder and reads the data from the cdc console of the sites configured
+  build [options]   Processes the local data and prepares it to be deployed to the cdc console
+  deploy [options]  Deploys the local data to the cdc console on the sites configured
+  help [command]    display help for command
+```
+
+### How to use filters on preview
+
+The filter argument allows the user to filter the screens he wants to see using the apiKeys that are configured on the configuration file cdc-accelerator.json, for example:
+
+```
+ [{
+    apiKey: '1_2ABCDEFGHI345',
+    screens: [{ screenSetID: 'PreferencesCenter-ProfileUpdate', screenID: 'gigya-update-profile-screen' }],
+    emails: [ { emailID: 'codeVerification', languages: ['en'] } ]
+  }]
+```
+
+Here we are dealing with the apiKey '1_2ABCDEFGHI345', the filter is saying that the screen the user will see is the 'PreferencesCenter-ProfileUpdate' and the email template will be 'codeVerification' on that apiKey
+If the user wants to use the filter on more than one apiKey he can use the filter like this:
+
+```
+[{
+    apiKey: '1_2ABCDEFGHI345',
+    screens: [
+      { screenSetID: 'PreferencesCenter-ProfileUpdate', screenID: 'gigya-update-profile-screen' },
+      { screenSetID: 'PreferencesCenter-Landing', screenID: 'gigya-login-screen' },
+    ],
+    emails: []
+  },
+  {
+    apiKey: '1_3AS9DJAKSLA12',
+    emails: [{ emailID: 'doubleOptIn', languages: ['ar', 'en', 'pt-br'] } ]
+  }]
+```
+
+Here the first apiKey will have the webScreenSets filtered and on the second apiKey it will only show the emailTemplate 'doubleOptIn'.
+If the user needs to add that filter to all the ApiKeys that are on the configuration file, he can simply replace the apikey ID to '\*' for example:
```

Comment:
Remove the \ from '\*'
The filter with apiKey * is not for all api keys, is just for the other api keys that do not have already a specific filter configuration.
If there is not a filter applied to any api key then a filter for api key * is for all api keys